src/resmom/mom_hook_func.c

```diff
@@ -3081,17 +3079,17 @@ struct work_task *pwt;
 void send_hook_fail_action(hook *phook)
 {
 
-	char	hook_buf[HOOK_BUF_SIZE];
+	char hook_buf[HOOK_BUF_SIZE];
 	vnl_t *tvnl = NULL;
-	int	vret = -1;
+	int vret = -1;
 
 	if ((phook == NULL) || (phook->hook_name == NULL)) {
 		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_HOOK,
 			  LOG_ERR, __func__, "Hook received is NULL");
 		return;
 	}
 
-	snprintf(hook_buf, sizeof(hook_buf)-1, "1,%s", phook->hook_name);
+	snprintf(hook_buf, sizeof(hook_buf) - 1, "1,%s", phook->hook_name);
```

Comment:
No need for the sizeof() -1.  snprintf() will handle the null byte. (here and other places)

Reply:
this was existing code - I just added spaces. should this be taken care of in this PR?

Reply:
Up to you.  I guess if it is existing, then there is more cases than just the few your modified.

Reply:
I changed the ones I could find :).